Sources/Datadog/FeaturesIntegration/CrashReporting/CrashReportingWithRUMIntegration.swift

```diff
@@ -18,7 +18,7 @@ internal struct CrashReportingWithRUMIntegration: CrashReportingIntegration {
         static let viewEventAvailabilityThreshold: TimeInterval = 14_400 // 4 hours
     }
 
-    /// The output for writting RUM events. It uses the authorized data folder and is synchronized with the eventual
+    /// The output for writing RUM events. It uses the authorized data folder and is synchronized with the eventual
```

Comment:
Good catch!

Reply:
I found this option on Xcode 🔨 

<img width="664" alt="Screenshot 2021-03-04 at 10 49 39" src="https://user-images.githubusercontent.com/2358722/109945233-558c4100-7cd7-11eb-8589-f8af462f805b.png">